bpf/process/types/basic.h

```diff
@@ -2286,6 +2296,13 @@ do_actions(void *ctx, struct selector_action *actions, struct generic_maps *maps
 {
 	bool post = true;
 	__u32 l, i = 0;
+	int zero = 0;
+	struct msg_generic_kprobe *e;
+
+	e = map_lookup_elem(maps->heap, &zero);
+	if (!e)
+		return 0;
+	e->post = true;
```

Comment:
I wonder we could do this directly in the lsm code code.. I think generic_actions could return post/nopost value
and lsm core code could use it to set e->post ... and all the other generic sensors could just ignore the generic_actions return value and return 0, which is what they do now anyway, wdyt?

Reply:
Yes, I think, it's a good idea, I'll try!